docs/api/domains.md

```diff
@@ -135,5 +135,25 @@ const Planets = {
   }
 }
 
-repo.push(Actions.add, { name: 'earth' }) // this will add Earth
+repo.push(addPlanet, { name: 'earth' }) // this will add Earth
+```
+
+Registration is given the command associated with an action, and the
+current status of the action. Additionally, functions returned from
+register will be used as the handler for a Microcosm dispatch:
+
+```javascript
+import { addPlanet } from '../actions/planets'
+
+const StatusLog = {
+  //...
+  register (command, status) {
```

Comment:
For effects, every time. For domains, we cache the answer. So there's actually an edge case here - we should only cache on the old version of register, or not cache at all. Otherwise, I think the expectation would be that this gets called every single time, when it wouldn't for Domains (because of caching).

Hmmm. This feature is really for effects, not Domains. What do you think about only doing this for Effects? Is the inconsistency there confusing?